src/pages/academia.js

```diff
@@ -42,6 +43,39 @@ const StyledButton = styled(Button)`
     margin-top: ${size.large};
 `;
 
+const BodyContent = styled('div')`
+    margin: 0 auto;
+    max-width: ${CONTENT_MAX_WIDTH};
+    width: 100%;
+`;
+
+const OfferingsContent = styled('div')`
+    margin-bottom: ${size.xlarge};
+    max-width: 324px;
```

Comment:
I have a feeling when we put images in this will change but I am fine leaving this like so for now. Likely we will use something like css grid or flexbox to handle responsiveness with the side by side layout.